book/03-hacking-atom/sections/02-init-file.asc

```diff
@@ -1,7 +1,7 @@
 [[_the_init_file]]
 === The Init File
 
-When Atom finishes loading, it will evaluate `init.coffee` in your `~/.atom` directory, giving you a chance to run CoffeeScript code to make customizations. Code in this file has full access to https://atom.io/docs/api/latest/Atom[Atom's API]. If customizations become extensive, consider creating a package, which we will cover in <<_package_word_count>>.
+When Atom finishes loading, it will evaluate `init.coffee` in your `~/.atom` directory, giving you a chance to run CoffeeScript code to make customizations. Code in this file has full access to https://atom.io/docs/api/latest/AtomEnvironment[Atom's API]. If customizations become extensive, consider creating a package, which we will cover in <<_package_word_count>>.
```

Comment:
If we change the link to `https://atom.io/docs/api/latest/`, it will redirect to the proper location and save us from updating this if the name of the overarching class changes again.